packages/dev/core/src/Rendering/prePassRenderer.ts

```diff
@@ -418,6 +418,17 @@ export class PrePassRenderer {
         }
     }
 
+    public setDepthPeelingOutput(rt: RenderTargetTexture) {

```

Comment:
I think we should not use the word `DepthPeeling` in the pre pass renderer, as it should remain as neutral as possible regarding the graphical sub components.

Maybe `setCustomOutput`?